spec/run_oc_pedant.rb

```diff
@@ -104,7 +146,8 @@ def start_cheffs_server(chef_repo_path)
     # are turned off" - @jkeiser
     #
     # ...but we're not there yet
-    '--skip-keys',
+    '--skip-controls',
+    '--skip-acl',
```

Comment:
I'm not sure. Maybe a bad rebase, or maybe I added them temporarily and forgot to take them out. I'll do that now.